text/0000-sync-networked-vars.md

```diff
@@ -0,0 +1,110 @@
+- Feature Name: `synced_networked_var`
+- Start Date: 2021-02-12
+- RFC PR: [Unity-Technologies/com.unity.multiplayer.rfcs#0000](https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/pull/0000)
+- Issue: [Unity-Technologies/com.unity.multiplayer#0000](https://github.com/Unity-Technologies/com.unity.multiplayer/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Make the NetworkedVariables synchronized with the tick system.
+
+We suggest two elements:
+- Changing `NetworkedBehaviour::NetworkedBehaviourUpdate()` so that the function runs at most one per Network Tick. When called more than once per Network Tick, it would simply return.
```

Comment:
I think this part could be further explained: why we're making this change? etc.

Reply:
perhaps, maybe outline the specific network update flow leads into network variable set (write) and get (read) in the reference-level explanation section below?

Reply:
Added diagram showing the send-side write of ticks. Receive-side will be coming shortly.

Reply:
Worth mentioning that today NetworkedBehaviourUpdate runs based on a custom-configurable arbitrary time not a proper coordinated tick